src/view/shell/index.tsx

```diff
@@ -80,19 +81,9 @@ function ShellInner() {
     }
   }, [currentAccount])
 
-  /**
-   * The counterpart to `accessibilityViewIsModal` for Android. This property
-   * applies to the parent of all non-modal views, and prevents TalkBack from
-   * navigating within content beneath an open dialog.
-   *
-   * @see https://reactnative.dev/docs/accessibility#importantforaccessibility-android
-   */
-  const importantForAccessibility =
-    openDialogs.length > 0 ? 'no-hide-descendants' : undefined
-
   return (
     <>
-      <View
+      <Animated.View
```

Comment:
Super clever I love it

Reply:
Yea, it's bizarre to think about at first as to why it works, but the gist is that the `SharedValue` creates a reference on the native side that can be updated synchronously, and therefore the underlying `UIView` (or whatever it is on android) can just update on the native side rather than causing a React re-render. Not something we want to do very often obviously, but in situations like this it's pretty useful.